base/interactiveutil.jl

```diff
@@ -251,7 +251,8 @@ function versioninfo(io::IO=STDOUT, verbose::Bool=false)
         println(io, "DEBUG build")
     end
     println(io,             "Platform Info:")
-    println(io,             "  System: ", Sys.KERNEL, " (", Sys.MACHINE, ")")
+    println(io,             "  OS: ", is_windows() ? "Windows" : is_apple() ?
```

Comment:
For detecting the Linux distro, it seems like [checking for an /etc/*release*](http://unix.stackexchange.com/questions/35183/how-do-i-identify-which-linux-distro-is-running) file may be easiest; in systemd distros, `/etc/os-release` is standard.